### PR TITLE
metrics, pd_worker: add time duration metrics for the Load Base Split

### LINKED ISSUE #12937: Record the time consumed by Load Base Split
## Development Task

Due to some statistical and computational work, it is necessary to record the time consumed by Load Base Split, which can help us to troubleshoot its potential performance problems.

### PR DESCRIPTION
**Pull request by JmPotato**
Signed-off-by: JmPotato <ghzpotato@gmail.com>

### What is changed and how it works?

Issue Number: Close #12937.

What's Changed:

```commit-message
Add time duration metrics for the Load Base Split.
```

### Check List 

Tests 

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)

image

### Release note 

```release-note
None.
```

### CODE REVIEW

**Review by lhy1024 — APPROVED** [review]

File: metrics/grafana/tikv_details.json [hunk 1]
```diff
@@ -15476,6 +15476,132 @@
...
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "stack": false,
+          "steppedLine": false,
+          "targets": [
+            {
+              "exemplar": true,
+              "expr": "histogram_quantile(0.80, sum(rate(tikv_load_base_split_duration_seconds_bucket{tidb_cluster=~\"$tidb_cluster.*\", instance=~\"$instance\"}[1m])) by (le, instance))",
```
Comment (rleungx) [inline metrics/grafana/tikv_details.json:15534]: Do we need `k8s_cluster=\"$k8s_cluster\"`?

Comment (JmPotato) [reply metrics/grafana/tikv_details.json:15534]: It's introduced by https://github.com/tikv/tikv/pull/12008.

Comment (rleungx) [reply metrics/grafana/tikv_details.json:15534]: Yes, seem this PR doesn't add it.

Comment (JmPotato) [reply metrics/grafana/tikv_details.json:15534]: Oops, I will add it back.

**Review by nolouch — APPROVED** [review]

**Review by disksing — APPROVED** [review]

**Pull request merged**